CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ that users understand how the changes affect the new version.
 
 development
 ---------------------------
++ Small changes were made to support miniwdl and miniwdl is not the recommended
+  tool to run this pipeline.
```

Comment:
This is somewhat conflicting with what is said in the docs.
> For local and HPC usage we recommend using miniwdl.

Reply:
Typo, should be `now` I will change it. Good catch!